src/os_io_seproxyhal.c

```diff
@@ -1004,8 +1004,9 @@ void io_seproxyhal_power_off(bool criticalBattery)
         (char) criticalBattery,
     };
     io_seproxyhal_spi_send(seph_io_device_off, sizeof(seph_io_device_off));
-    for (;;)
+    for (;;) {
         ;
+    }
```

Comment:
Can tou drop this change that isn't link with this PR and might make the cherry-pick process more difficult.

Reply:
good catch thanks